<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioModTx</class>
 <widget class="QMainWindow" name="RadioModTx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radio Modulation Transmitter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="DatasetGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Dataset</string>
    </property>
    <widget class="QRadioButton" name="DatasetRadioML2016RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioML 2016.10A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DatasetRadioML2018RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioML 2018.01</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DatasetHisarMod2019RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>HisarMod 2019.1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenDatasetButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FramesGroupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>181</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Frames generation</string>
    </property>
    <widget class="QPushButton" name="StartFramesButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopFramesButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FramesTxComboBox">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>40</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="FramesTxLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="FloSpinBox">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>80</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="suffix">
      <string> MHz</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>70.000000000000000</double>
     </property>
     <property name="maximum">
      <double>6000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>2400.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="FloLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f&lt;span style=&quot; vertical-align:sub;&quot;&gt;LO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="BwLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>BW =</string>
     </property>
    </widget>
    <widget class="QLabel" name="BwValue">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>40.000 MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="GainLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>G =</string>
     </property>
    </widget>
    <widget class="QLabel" name="GainValue">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="FsampLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f&lt;span style=&quot; vertical-align:sub;&quot;&gt;S&lt;/span&gt; =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="FsampValue">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>61.440 MHz</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="NcoGainSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="NcoGainLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NCO gain&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ModulationGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Modulation</string>
    </property>
    <widget class="QComboBox" name="ModulationSnrComboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ModulationSnrText">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SNR =</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ModulationNameComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ModulationTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Type :</string>
     </property>
    </widget>
    <widget class="QLabel" name="ModulationFamilyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Family :</string>
     </property>
    </widget>
    <widget class="QLabel" name="ModulationTypeValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
    <widget class="QLabel" name="ModulationFamilyValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>DatasetRadioML2016RadioButton</tabstop>
  <tabstop>DatasetRadioML2018RadioButton</tabstop>
  <tabstop>DatasetHisarMod2019RadioButton</tabstop>
  <tabstop>OpenDatasetButton</tabstop>
  <tabstop>ModulationNameComboBox</tabstop>
  <tabstop>ModulationSnrComboBox</tabstop>
  <tabstop>FramesTxComboBox</tabstop>
  <tabstop>FloSpinBox</tabstop>
  <tabstop>NcoGainSpinBox</tabstop>
  <tabstop>StartFramesButton</tabstop>
  <tabstop>StopFramesButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
